### ISSUE TITLE
Adding the newly created PR to a project

### ISSUE THREAD

**Issue by ivanyv** [devops-infra/action-pull-request#133]
Would it be too hard or out of scope to add this feature?

**Comment by ChristophShyper** [comment]
Hi @ivanyv .
I'll have to look some day at the API and check it.

**Comment by ChristophShyper** [comment]
Hello @ivanyv 
You're probably not interested anymore, but I finally have some time and willingness to resolve all pending issues.
Project can be now passed as input https://github.com/devops-infra/action-pull-request#:~:text=Milestone-,project,-No

**Comment by ivanyv** [comment]
@ChristophShyper right, the workflow doesn't require this anymore, but I'm sure it can be helpful to others, thanks!
